### DISCUSSION TITLE
IOS settings.

### DISCUSSION THREAD

**Discussion by ghost** [the-weird-aquarian/privacy-settings#1] [Show and tell]
Hi!

First of all, very good work. You can be proud of yourself.
As I'm very interested in confidentiality, I'd like to collaborate with you by sharing my IOS settings with you.

**Comment by StellarSand** [comment]
Hello, 

> First of all, very good work. You can be proud of yourself.

Thank you. Glad you found this guide helpful. :)

> As I'm very interested in confidentiality, I'd like to collaborate with you by sharing my IOS settings with you.

Yes this would be very much appreciated. Feel free to submit a pull request [here](https://github.com/the-weird-aquarian/privacy-settings/pulls).
